shell.nix

```diff
@@ -11,7 +11,7 @@ let
 in pkgs.lib.overrideDerivation iohk-ops.env
    (old: {
      shellHook = ''
-       io () {
+       function io {
```

Comment:
@deepfire what's the reason for this change? If I understand correctly `io ()` is POSIX.

Reply:
i also ran into this failing on the deployer, couldnt figure out why it was a problem

Reply:
Can you paste the error?

Reply:
Aha was it:

```
$ nix-shell
bash: eval: line 69: syntax error near unexpected token `('
bash: eval: line 69: `io () {'
``

Reply:
yep, exactly

Reply:
Really strange, bash should support this syntax.